stdlib/header.c

```diff
@@ -30,7 +30,10 @@
 #include "caml/mlvalues.h"
 #include "caml/exec.h"
 
-char * default_runtime_path = RUNTIME_NAME;
+#define Q(x) #x
+#define QUOTE(x) Q(x)
+
+char * default_runtime_path = QUOTE(RUNTIME_NAME);
```

Comment:
This looks like magic, and in particular it is not at all obvious why you need to define two different macros instead of just on. Maybe a small comment to point out that yes, this is intentional, could avoid people trying to "simplify" the code in the future by inlining the definition of QUOTE?

Reply:
Maybe just pointing to this link in comments in the code would be helpful.

Reply:
Indeed - we do a similar trick in `CAML_WIDEN_STRING_LITERAL` in `misc.h` - the extra comment certainly doesn't hurt (I specifically describe the chapter, as misc.h cites an MSDN source which now no longer exists!)

Reply:
Hah - now I've managed to push, but there's a [reported incident with pull requests](https://www.githubstatus.com/incidents/1g1gkh0qpyvs) 😂

Reply:
Time to call it a day...